mars/sorting.lfr

```diff
@@ -0,0 +1,12 @@
+module sorting (
+    finput A,
+    foutput a_out, b_out, c_out
+);
+
+    // sorter
+    #MAP 'Sorter' '~'
+    assign a_out = ~A;
+    assign b_out = ~A;
+    assign c_out = ~A;
```

Comment:
This needs to be `a_out = A - {b_out, c_out}`

Reply:
so b_out = A - {a_out, c_out} and so on?